using_images/db_images/mysql.adoc

```diff
@@ -62,119 +38,42 @@ This image is available on DockerHub. To download it:
 `$ docker pull openshift/mysql-55-centos7`
 ****
 
-Alternatively, you can build a CentOS 7 based MySQL image:
-
-. Clone the following repository:
-+
-****
-`$ git clone https://github.com/openshift/mysql.git`
-****
-. Navigate into the directory you just created:
-+
-****
-`$ cd mysql`
-****
-. Build the image:
-+
-****
-`$ make build VERSION=5.5`
-****
-
-[NOTE]
-====
-If you omit the `VERSION` parameter, the build action is performed on all
-the provided versions of MySQL. Since version `5.5` is the only verision provided,
-you can omit this parameter.
-====
+To use these images, you can either access them directly in those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
 
-== Usage
+== Configuration and Usage
 
-To create a container running MySQL, you must run one of following commands,
-depending on your base image.
+=== Initializing the Database
 
-*RHEL 7 Base Image*
+The first time you use the shared volume and initialize the database, the
```

Comment:
I feel that this sentence implies that the user has to do some initialization, which isn't strictly true since our entrypoint script really does that.